red-hat-storage/cephci
# subvolume metatdata validation

pre-requisites:
    1. Subvolume created
    2. A set of test candidates for metadata
Test operation:
    1. Create subvolume
    2. Set the metadata from the set
    3. Try to set the metadta with same key
    4. Try to remove the metadata key
    5. Check if the metadata key and value are deleted
Log:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-49SQ84
 





Signed-off-by: julpark-rh <yonhyun@gmail.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

</details>
